TaskWithNullptr/TaskWithNullptr.cpp

```diff
@@ -0,0 +1,16 @@
+﻿#include <iostream>
+#include <excpt.h>
+int main()
+{
+	__try
+	{
+		double* ptr = nullptr; // create a null pointer 
+		std::cout << *ptr; // dereference a null pointer, exception will trigger
+	}
+	__except (EXCEPTION_EXECUTE_HANDLER)
+	{
+		std::cout << "It is nullptr" << std::endl;
```

Comment:
Obviously, we'd better return -1 in the __except block. We will improve in further tasks.